apps/browser/src/autofill/background/notification.background.ts

```diff
@@ -163,14 +168,36 @@ export default class NotificationBackground {
       activeUserId,
     );
 
+    const organizations = await firstValueFrom(
+      this.organizationService.organizations$(activeUserId),
+    );
+
     return decryptedCiphers.map((view) => {
-      const { id, name, reprompt, favorite, login } = view;
+      const { id, name, reprompt, favorite, login, organizationId } = view;
+
+      const organizationType = organizationId
+        ? organizations.find((org) => org.id === organizationId)?.productTierType
+        : null;
+
+      const cipherIndicatorIcons: CipherIndicatorIconType[] = [];
```

Comment:
hrm, maybe `organizationCategory`? The icon concept is a presentational concern, not directly relevant to the data that drives the state.

```suggestion
      const organizationCategory: OrganizationCategory[] = [];
```

Reply:
^ not organization "type" since that's ostensibly what `productTierType` is

Reply:
Ah, I agree. Makes sense, great callout

Reply:
I went with `const organizationCategories: OrganizationCategory[] = [];` since its an array